README.md

```diff
@@ -16,6 +16,13 @@ as well as deobfuscate scripts processed by
 # Usage
 You can beautify javascript using JS Beautifier in your web browser, or on the command-line using node.js or python.
 
+To use in web browser include the script tag below in your document
+```html
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify.js"></script>
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-css.js"></script>
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-html.js"></script>
```

Comment:
It looks like you'll need to use a specific version and add something to the release.sh script to update this.

https://github.com/rgrove/rawgit/wiki/Frequently-Asked-Questions#how-long-does-the-cdn-cache-files-how-can-i-make-it-refresh-my-file 

Also, the cdn in not guaranteed to be up.  Please add a disclaimer of some kind or at least point to the FAQ so people understand the limits of this functionality.